### PR TITLE
Fix #2671 on GitHub

### LINKED ISSUE #2671: Bazel chocolatey installation error
I want to install Bazel on Windows 7 64bit. I am using Chogolatey v0.10.3, and then I ran "choco install bazel", and got an error: Method invocation failed because [System.Object[]] doesn't contain a method named 'Split'. 

How can I install Bazel. I have tried to use the binary distribution with no success already. Thanks a lot.

### PR DESCRIPTION
**Pull request by petemounce**

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: scripts/packages/chocolatey/.gitignore [hunk 1]
```diff
@@ -2,5 +2,5 @@
 *.zip
 bazel.nuspec
 tools/*.exe
-tools/LICENSE
-tools/params.json
+tools/LICENSE*
```
Comment (petemounce) [inline scripts/packages/chocolatey/.gitignore]: These are unrelated drive-by fixes.

File: scripts/packages/chocolatey/build.ps1 [hunk 2]
```diff
@@ -33,10 +33,10 @@ if ($fixPackage -eq $true) {
   }
   $tvPackageFixVersion = "$($prefix)$((get-date).tostring("yyyyMMdd"))"
 }
-rm -force -ErrorAction SilentlyContinue ./*.nupkg
-rm -force -ErrorAction SilentlyContinue ./bazel.nuspec
-rm -force -ErrorAction SilentlyContinue ./tools/LICENSE
-rm -force -ErrorAction SilentlyContinue ./tools/*.orig
+rm -force -ErrorAction SilentlyContinue "./*.nupkg"
+rm -force -ErrorAction SilentlyContinue "./bazel.nuspec"
+rm -force -ErrorAction SilentlyContinue "./tools/LICENSE*"
```
Comment (petemounce) [inline scripts/packages/chocolatey/build.ps1]: These are unrelated drive-by fixes.

File: scripts/packages/chocolatey/tools/chocolateyinstall.ps1 [hunk 3]
```diff
@@ -3,16 +3,18 @@ $packageName = 'bazel'
 
 $toolsDir = Split-Path -parent $MyInvocation.MyCommand.Definition
 $paramsText = get-content "$($toolsDir)\params.txt"
```
Comment (petemounce) [inline scripts/packages/chocolatey/tools/chocolateyinstall.ps1]: `get-content` already returns an array of strings, but it sounds like #2671 problem is that powershell v2 treats them as `system.object` rather than `system.string`. Which is super nice.

**Comment by petemounce** [comment]
I've also pushed https://chocolatey.org/packages/bazel/0.4.5-rc7-20170314 so that RC has the same fix.

**Comment by meteorcloudy** [comment]
LGTM. thanks!

**Comment by meteorcloudy** [comment]
I'll import this

**Comment by petemounce** [comment]
Wonderful, thanks.
